R/FilterStateDatettime.R

```diff
@@ -116,19 +116,10 @@ DatetimeFilterState <- R6::R6Class( # nolint
       checkmate::assert_multi_class(x, c("POSIXct", "POSIXlt"))
       super$initialize(x, varname, varlabel, dataname, extract_type)
 
-      var_range <- range(x, na.rm = TRUE)
+      var_range <- as.POSIXct(round(range(x, na.rm = TRUE), units = "secs"))
       private$set_choices(var_range)
       self$set_selected(var_range)
 
-      if (shiny::isRunning()) {
-        session <- getDefaultReactiveDomain()
-        if (!is.null(session$userData$timezone)) {
-          private$timezone <- session$userData$timezone
-        }
-      } else if (isTRUE(attr(x, "tz") != "")) {
-        private$timezone <- attr(x, "tz")
-      }
```

Comment:
Why to remove this one? Warnings reported by @asbates few lines below are related to this? @asbates what happens if you add these lines back?

Reply:
Because I understand this stores the timezone of the local machine, which is redundant.

Reply:
Sorry, I misspoke. Timezone of the data is what we want, do we not?

Reply:
I agree.

Reply:
Definitely. This makes more sense as another issue.